src/index.ts

```diff
@@ -56,24 +66,59 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
-function addComment(crawlerId: string): void {
-  try {
-    const pathArray = CRAWLER_API_BASE_URL.split('/');
-    const protocol = pathArray[0];
-    const host = pathArray[2];
-    const baseUrl = `${protocol}//${host}`;
+function findCommentPredicate(crawlerId: string, comment: Comment): boolean {
+  return comment.body ? comment.body.includes(crawlerId) : true;
```

Comment:
```suggestion
  return comment.body ? comment.body.includes(crawlerId) : false;
```
seems more correct, otherwise you always return the first comment without body